app/src/main/res/layout/item_edit_history.xml

```diff
@@ -57,6 +59,7 @@
             android:paddingBottom="8dp"
             android:textAlignment="viewStart"
             android:textAllCaps="false"
+            app:rippleColor="@null"
```

Comment:
If you would like to make this "button" unclickable, does it mean it should be a `Textview`?

Reply:
It is easier to style the rounded edges @cooltey but I could probably not use the style Flatbutton.Small and just have it inline. Do you feel that is better?

Reply:
Makes sense to me. Or, maybe we can use `rounded_12dp_corner_base90_fill` drawable to achieve this since making a `Button` not clickable also with some other extra setting to the view does not seem right to me. How do you think @dbrant?

Reply:
Did this @cooltey